src/Behat/FeatureContext.php

```diff
@@ -37,13 +52,21 @@ public function theCurrentVersionsAreShown()
     }
 
     /**
-     * @Given I log in as :user with password :password
+     * @Given I log in as :user with :password
      */
     public function iLogInAs($user, $password)
```

Comment:
I think we should split this into two functions as having `$password` as a variable to use in the BDD makes it look like the person writing the feature test can enter an actual password rather than just control if the correct password is used or not 

```suggestion
     /**
     * @Given I log in as :user with the correct password
     */
    public function iLogInAsWithCorrectPassword($user)
    {
    // code for correct password entered
    }
    
         /**
     * @Given I log in with the incorrect password
     */
    public function iLogInWithIncorrectPassword()
    {
    // code for incorrect password entered
    }
```